base/doc.dtx

```diff
@@ -2201,7 +2201,7 @@
 % \changes{v1.7a}{1992/03/13}{Added \cs{reset@font} for NFSS.}
 % \changes{v1.8c}{1993/10/25}{NFSS standard}
 % \changes{v1.9t}{1995/05/26}{Removed \cs{math@fontsfalse} (different
-%                             math setup /pr1622}
+%                             math setup /pr1622)}
```

Comment:
Maybe
```suggestion
%                             math setup gh/1622)}
```

Reply:
in 1995 we haven't been on github :-) this must refer to the old latex bug db

Reply:
Ah it was 1995! You're right.